RMQ/rabbitMQ.js

```diff
@@ -4,19 +4,21 @@ const serverLogger = require(`../logger`);
 const logger = serverLogger.log;
 
 const sendQueueUpdate = (user_mail, credits) => {
+
     amqp.connect(amqpURL, (err, conn) => {
-        conn.createChannel((err, ch) => {
+        conn.createChannel(async (err, ch) => {
             const q = 'CloudAMQP';
-            const msg = {
+            const subscriptionDetails = {
                 "user_mail": {user_mail},
                 "credits": {credits}
             };
-            const stringMsg = JSON.stringify(msg);
+            const stringMsg = JSON.stringify(subscriptionDetails);
             ch.assertQueue(q, {durable: false});
-            setInterval(() => {
-                ch.sendToQueue(q, Buffer.from(stringMsg))
-                    .then(logger.info(`RMQ message sent: ${msg}`));
-            })
+            const sendQueue = async () => {
+                ch.sendToQueue(q, Buffer.from(stringMsg));
+            };
+            await sendQueue().then(() => logger.info(`RMQ- update free plan message sent to user: ${user_mail} with: ${credits} credits`));
```

Comment:
It you write 'free' static, so this function is for the cron job only, right? 
we'll add another function to the pro/premium